src/source/temporalgrid_tile_worker_source.js

```diff
@@ -33,6 +33,9 @@ const getAggregationparams = params => {
     if (url.searchParams.get("interval")) {
         aggregationParams.interval = url.searchParams.get("interval")
     }
+    if (url.searchParams.get("breaks")) {
```

Comment:
Not exactly related with this PR but I think we should follow the same for the `singleFrame` and `serverSideFilters` params, I mean to only include them in the url when needed so the API cache is used.
```
if (singleFrame !== undefined) {
  newUrl.searchParams.append('temporal-aggregation', singleFrame);
}
```
```
const serverSideFilters = originalUrl.searchParams.get("serverSideFilters")
    if (serverSideFilters) {
        newUrl.searchParams.append('filters', encodeURIComponent(serverSideFilters));
    }
```